UM/Qt/qml/UM/BurgerButton.qml

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2022 Ultimaker B.V.
+// Uranium is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Controls 2.3
+
+import UM 1.5 as UM
+
+Button
+{
+    id: control
+    padding: 0
+
+    background: Item {}
+
+    contentItem: Rectangle
```

Comment:
Why not just use the hover circle as the background and keep the RecolorImage as the content item? That makes a bit more conceptual sense to me.